op-mode-definitions/show-hardware.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="show">
+    <children>
+      <node name="hardware">
+        <properties>
+          <help>Show system hardware details</help>
+        </properties>
+        <children>
+          <node name="cpu">
+            <properties>
+              <help>Show CPU info</help>
+            </properties>
+            <command>lscpu</command>
+            <children>
+              <node name="detail">
+                <properties>
+                  <help> Show system CPU details</help>
+                </properties>
+                <command>less $_vyatta_less_options --prompt='.cpu info' -- /proc/cpuinfo</command>
+              </node>
+              <node name="summary">
+                <properties>
+                  <help>Show CPU's on system</help>
+                </properties>
+                <command>${vyatta_bindir}/vyatta-cpu-summary.pl</command>
```

Comment:
I've made a replacement for the old CPU summary script, it will be in ${vyos_op_scripts_dir}/cpu_summary.py
Since the format is common in /proc, I've made vyos.util.colon_separated_to_dict function for that.